site/documentation/static/api/device-registry-v1.yaml

```diff
@@ -486,6 +486,10 @@ components:
                description: The Base64 encoded binary DER encoding of the
                             trusted root certificate. Either this property
                             or `public-key` must be set.
+            "algorithm":
+               type: string
+               description: The algorithm of the key material.
```

Comment:
I think this can use some more detail:
> The algorithm used for the public key of the CA. If the "cert" property is used to provide an X.509 certificate then the algorithm is determined from the certificate and this property is ignored. Otherwise, i.e. if the "public-key" property is used, this property must be set to the algorithm used if other than "RSA" (the default value).